ouroboros-consensus-cardano/src/unstable-cardano-testlib/Test/Consensus/Cardano/Examples.hs

```diff
@@ -75,28 +77,83 @@ eraExamples =
 combineEras ::
      NP Examples (CardanoEras Crypto)
   -> Examples (CardanoBlock Crypto)
-combineEras = mconcat . hcollapse . hap eraInjections
+combineEras perEraExamples = Examples {
```

Comment:
It's a pity this has now become so unwieldy.

Reply:
If you want to get it reworked I'd rather do it on a follow-up than to complicate this PR.